disnake/app_commands.py

```diff
@@ -437,13 +437,20 @@ class ApplicationCommand(ABC):
         Defaults to ``True``.
 
         .. versionadded:: 2.5
+
+    nsfw: :class:`bool`
+        Whether this command can only be used in NSFW channels.
+        Defaults to ``False``.
```

Comment:
we might be able to merge this without dm support but we do not yet know how DMs will work.

Reply:
They don't show those commands in dms